cmake/modules/DefineCompilerFlags.cmake

```diff
@@ -16,7 +16,8 @@ if (${CMAKE_C_COMPILER_ID} MATCHES "(GNU|Clang)")
     # cannot be pedantic with sqlite3 directly linked
     # FIXME Can we somehow not use those flags for sqlite3.* but use them for the rest of csync?
     if (NOT USE_OUR_OWN_SQLITE3)
-        set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=gnu99 -pedantic -pedantic-errors")
+        # -pedantic-errors explodes in src/csync/csync_private.h:166
```

Comment:
csync_private.h:166   is a comment for me.
So I guess this comment is already outdated and should be removed.

Maybe pedentic-errors would make sens on a CI, but not when building packages, so it is fine to remove it.

Reply:
Ooh of course, it was early, when I wrote the comment ... 😄 
Yeah, let's just remove it.